nifi-assembly/pom.xml

```diff
@@ -445,6 +445,24 @@ language governing permissions and limitations under the License. -->
             <version>1.6.0-SNAPSHOT</version>
             <type>nar</type>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-elasticsearch-client-service-api-nar</artifactId>
```

Comment:
My build is failing due to this. I don't see this artifact created anywhere.

Reply:
Weird, I thought that I took that out.